src/UI/class-recommended-widget.php

```diff
@@ -350,28 +350,16 @@ private function get_boost_params(): array {
 	}
 
 	/**
-	 * Checks if both the API key and API secret settings are populated with
+	 * Checks if both the Site ID and API secret settings are populated with
 	 * non-empty values.
 	 *
 	 * @since 2.5.0
 	 *
-	 * @return bool True if apikey and api_secret settings are not empty strings.
+	 * @return bool True if site_id and api_secret settings are not empty strings.
```

Comment:
Maybe add `apikey` here since we're referring to the setting itself?

```suggestion
	 * @return bool True if apikey and api_secret settings are not empty strings.
```